.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -270,21 +289,60 @@ jobs:
           }'
           wait-for-completion: false
 
-  notify-slack-on-failure:
+  notify-slack:
     runs-on: ubuntu-latest
     needs:
       - init-github-context
       - rawls-build-tag-publish-job
       - create-bee-workflow
       - rawls-swat-e2e-test-job
       - destroy-bee-workflow
-    if: ${{ failure() }} # Want to notify regardless of which step fails
+    if: always()
```

Comment:
Made this all the same as how we notify for the billing project test. QA will want to see successes as well as failures when it runs on promotion from testing.